blockchain/pool.go

```diff
@@ -0,0 +1,48 @@
+package blockchain
+
+// Pool is a set of valid Transactions.
+type Pool struct {
+	Miner        Address
+	Transactions map[Hash][]*Transaction
+}
+
+// NewPool initializes a new pool.
+func NewPool(miner Address) *Pool {
+	return nil
+}
+
+// Len returns the number of transactions in the Pool.
+func (p *Pool) Len() int {
+	return 0
+}
+
+// GetTransaction returns the transctions with input transaction Hash h.
+func (p *Pool) GetTransactions(h Hash) []*Transaction {
+	return nil
+}
+
+// PutTransaction inserts a transaction into the pool, returning
+// true if the Transaction was inserted (was valid).
+func (p *Pool) PutTransaction(t *Transaction) bool {
```

Comment:
Hmm, I still need to put a transaction into the `Pool` once it pops out of the channel, no?